### PR TITLE
Use custom ParseDuration

### PR DESCRIPTION
**Pull request by ntk148v**
ParseDuration parses a string into a time.Duration,
assuming that a year always has 365d, a week always
has 7d, and a day always has 24h.

### CODE REVIEW

**Comment by ntk148v** [comment]
Close: #83

File: pkg/common/utils.go [hunk 1]
```diff
@@ -233,3 +234,10 @@ func ReachableTCP(addr string) error {
 	_, err := net.DialTimeout("tcp", u.Host, 3*time.Second)
 	return err
 }
+
+// ParseDuration parses a string into a time.Duration, assuming that a year
+// always has 365d, a week always has 7d, and a day always has 24h.
+func ParseDuration(durationStr string) (time.Duration, error) {
+	duration, err := prommodel.ParseDuration(durationStr)
+	return time.Duration(duration), err
+}
```
Comment (vtdat) [inline pkg/common/utils.go]: Needs docs for what can be parse as Duration.

File: docs/note.md [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+- `d` - days
+- `w` - weeks
+- `y` - years
+
+For example, to set `autoscaling.interval` to 5 minutes: `interval: 5m`.
+
+Some invalid format examples:
+
+- `1h30m`
+- `5M`
```
Comment (vtdat) [inline docs/note.md]: I don't see `5M` in units.

Comment (vtdat) [reply docs/note.md]: ok my bad invalid

**Pull request merged**